.travis.yml

```diff
@@ -37,10 +41,10 @@ services: mongodb
 before_install:
     - PHP=$TRAVIS_PHP_VERSION
     # Matrix lines for intermediate PHP versions are skipped for pull requests
-    - if [[ ! $deps && ! $PHP = ${MIN_PHP%.*} && $PHP != hhvm && $TRAVIS_PULL_REQUEST != false ]]; then deps=skip; skip=1; fi
+    - if [[ ! $deps && ! $PHP = ${MIN_PHP%.*} && ! $PHP = hhvm* && $TRAVIS_PULL_REQUEST != false ]]; then deps=skip; skip=1; fi
```

Comment:
for anyone wondering, using ``hhvm*`` for comparisons here is done to allow easily to test against LTS versions of HHVM too (not added in the matrix currently, but I used ``hhvm-3.12`` at some point during the PR and decided to keep the more flexible detection)